core/DataTable/Filter/ReplaceSummaryRowLabel.php

```diff
@@ -53,18 +53,18 @@ public function __construct($table, $newLabel = null)
      */
     public function filter($table)
     {
-        $rows = $table->getRows();
```

Comment:
This is just faster. Instead of iterating over all rows, we just get the summary row directly as the summary row would be otherwise most likely at the very end. Saves a lot of time